scene/resources/packed_scene.cpp

```diff
@@ -175,10 +175,18 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 			ERR_FAIL_COND_V_MSG(n.parent == -1, nullptr, vformat("Invalid scene: node %s does not specify its parent node.", snames[n.name]));
 			NODE_FROM_ID(nparent, n.parent);
 #ifdef DEBUG_ENABLED
-			if (!nparent && (n.parent & FLAG_ID_IS_PATH)) {
-				WARN_PRINT(String("Parent path '" + String(node_paths[n.parent & FLAG_MASK]) + "' for node '" + String(snames[n.name]) + "' has vanished when instantiating: '" + get_path() + "'.").ascii().get_data());
-				old_parent_path = String(node_paths[n.parent & FLAG_MASK]).trim_prefix("./").replace("/", "@");
-				nparent = ret_nodes[0];
+			if (n.owner >= 0) {
+				NODE_FROM_ID(nowner, n.owner);
```

Comment:
I am also unhappy with these changes to the packed scene and Node.

Reply:
Perhaps I'm just approaching this incorrectly. This is to catch any orphaned nodes of previously exposed nodes. Happy to take another pass at it if there's something particular you don't like.